gnuradio-runtime/lib/flat_flowgraph.cc

```diff
@@ -143,31 +142,37 @@ namespace gr {
 
     // limit buffer size if indicated
     if(grblock->max_output_buffer(port) > 0) {
-      //std::cout << "constraining output items to " << block->max_output_buffer(port) << "\n";
+      //GR_LOG_DEBUG (d_debug_logger, "constraining output items to " << block->max_output_buffer(port));
```

Comment:
Again, I thought it was for debugging purposes but now I see why `d_debug_logger` is there.